AdvancedTask/AdvancedTask/Steps/SearchSkillSteps.cs

```diff
@@ -0,0 +1,72 @@
+﻿using AdvancedTask.AssertHelpers;
+using AdvancedTask.Pages.Components.ProfileOverview;
+using AdvancedTask.Test_Model;
+using AdvancedTask.Utilities;
+
+namespace AdvancedTask.Steps
+{
+    public class SearchSkillSteps:BaseClass
+    {
+        SearchSkillComponent SearchSkillComponentObj;
+        SearchSkillAssertion SearchSkillAssertionObj;
+        ProfileTabComponents ProfileTabComponentsObj;
+
+        public SearchSkillSteps()
+        {
+            ProfileTabComponentsObj = new ProfileTabComponents();
+            SearchSkillComponentObj = new SearchSkillComponent();
+            SearchSkillAssertionObj = new SearchSkillAssertion();
+        }
+
+        public void SearchBySkill()
+        {
+            List<SearchSkill> SearchSkillData = JsonReader.ReadTestDataFromJson<SearchSkill>("A:\\Industry Connect\\AdvancedSprint1\\AdvancedTask\\AdvancedTask\\Json Test Data\\SearchSkill.json");
+           // ProfileTabComponentsObj.ClickSearchSkillIcon();
```

Comment:
remove all commented out code that is not a sentence, bad practice to keep commented out code as it confuses others

Reply:
Removed